pandas/core/strings.py

```diff
@@ -328,19 +328,42 @@ def str_contains(arr, pat, case=True, flags=0, na=np.nan, regex=True):
 
 def str_startswith(arr, pat, na=np.nan):
     """
-    Return boolean Series/``array`` indicating whether each string in the
-    Series/Index starts with passed pattern. Equivalent to
-    :meth:`str.startswith`.
+    Test if the start of each string element matches a pattern.
+
+    Return a Series of booleans indicating whether the given pattern matches
+    the start of each string element.
+    Equivalent to :meth: `str.startswith`.
 
     Parameters
     ----------
     pat : string
-        Character sequence
-    na : bool, default NaN
+        Character sequence.
+    na : object, default NaN
+        Character sequence shown if element tested is not a string.
 
     Returns
     -------
     startswith : Series/array of boolean values
+
+    Examples
+    --------
+    >>> s = pd.Series(['bat', 'bear', 'cat'])
+    >>> s.str.startswith('b')
+    0     True
+    1     True
+    2    False
+    dtype: bool
+    >>> s = pd.Series(['bat', 'bear', 'cat', np.nan])
+    >>> s.str.startswith('b', na='not_a_string')
+    0            True
+    1            True
+    2           False
+    3    not_a_string
+    dtype: object
+
+    See Also
+    --------
+    endswith : same as startswith, but tests the end of string
```

Comment:
Add `str.startswith`.

Reply:
If I'm not wrong, I think your `endswith` is linked by Sphinx to `pandas.endswith` which doesn't exist. I think the right way would be `Series.str.endswith`. Similar examples link to the Python standard library version, `str.startswith` (the one @TomAugspurger says).

If I'm not wrong, the descriptions in the `See Also` section should start with a capital letter and finish with a period.

Reply:
I can't seem to get a link, have tried:
str_endswith
Series.str_endswith
Series.str.endswith
Series.string.endswith
endswith
Series.str.str_endswith